bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/XMLFactory.java

```diff
@@ -84,23 +84,15 @@
     static {
         Canonicalizer.init();
         try {
-            if (Canonicalizer.getInstance(CANONICALIZER_OMIT_COMMENTS) == null) {
-                Canonicalizer.register(CANONICALIZER_OMIT_COMMENTS,
-                        "org.apache.axiom.c14n.impl.Canonicalizer20010315OmitComments");
-            }
-            if (Canonicalizer.getInstance(CANONICALIZER_WITH_COMMENTS) == null) {
-                Canonicalizer.register(CANONICALIZER_WITH_COMMENTS,
-                        "org.apache.axiom.c14n.impl.Canonicalizer20010315WithComments");
-            }
-            if (Canonicalizer.getInstance(CANONICALIZER_EXCL_OMIT_COMMENTS) == null) {
-                Canonicalizer.register(CANONICALIZER_EXCL_OMIT_COMMENTS,
-                        "org.apache.axiom.c14n.impl.Canonicalizer20010315ExclOmitComments");
-            }
-            if (Canonicalizer.getInstance(CANONICALIZER_EXCL_WITH_COMMENTS) == null) {
-                Canonicalizer.register(CANONICALIZER_EXCL_WITH_COMMENTS,
-                        "org.apache.axiom.c14n.impl.Canonicalizer20010315ExclWithComments");
-            }
-            canonicalizer = Canonicalizer.getInstance(CANONICALIZER_WITH_COMMENTS);
+            Canonicalizer.register("http://www.w3.org/TR/2001/REC-xml-c14n-20010315",
```

Comment:
I had to do this change to make it work. The original version was giving NPE. Please review this. @SupunS

Reply:
I had to do that change because with the new jvm runtime, now we have two XMLFactory classes, and both register the same set of canonicalizers twice. That was giving errors saying the same canonicalizer is already present.
Should we silently fail on `AlgorithmAlreadyRegisteredException`?